src/ArticleList.jsx

```diff
@@ -0,0 +1,26 @@
+const ArticleList = ({ articles }) => {
+  console.log(articles);
+  return (
+    <section className="articleList">
+      <h2 id="h2">Articles</h2>
+      <ul>
+        {articles.map((article) => {
+          return (
+            <li key={article.article_id}>
```

Comment:
If you'd like to get rid of the bullet point in a list element you can do this by in App css adding `list-style-type: none` as a css property to `<li>` elements.